test/functional/test-url-replacements.js

```diff
@@ -479,6 +479,12 @@ describes.sandboxed('UrlReplacements', {}, () => {
     });
   });
 
+  it('should replace TIMEZONE_CODE', () => {
+    return expandUrlAsync('?tz_code=TIMEZONE_CODE').then(res => {
+      expect(res).to.match(/tz_code=\w+/);
```

Comment:
Since `TIMEZONE_CODE` can resolved to empty string. We may need to reflect that in the test.